frontend/test/ui/test_homepage.py

```diff
@@ -58,8 +58,12 @@ def test_number_of_experiments(base_url, selenium):
     completed_experiments = len(
         [value for value in data['results'] if 'completed' in value and
          value['completed'] < str(datetime.utcnow())]) + 1
-    assert len(page.body.experiments) == int(
-        len(data['results']) - completed_experiments)
+    if page.featured.is_displayed:
+        assert len(page.body.experiments) == int(
+            len(data['results']) - completed_experiments - 1)
+    else:
+        assert len(page.body.experiments) == int(
+            len(data['results']) - completed_experiments)
```

Comment:
Can you add another assertion that the featured experiment specifically is not in the list of experiments?

Reply:
filed https://github.com/mozilla/testpilot/issues/3295, I still can't get integration tests running on my machine, it looks to be because of https://github.com/tox-dev/tox/issues/121. I can loop back around to get this test written when I'm back at my house and on my linux machine.